cypress/fixtures/elements.json

```diff
@@ -1,7 +1,7 @@
 {
     "appHeaderElements": {
```

Comment:
Am I correct in thinking that these selectors go in their own directory, for example `cypress/selectors/dashboard.json`? I only ask because I've been doing a fair amount of Cypress on CD (basing a lot of the structure off of Paul's old work).

I can't fully remember but I think the `selectors` directory was one of the defaults provided by Cypress.